public/assets/judges/judgesinfo.json

```diff
@@ -1,27 +1,27 @@
 [
     {
-        "name": "Judge1 LAST NAME",
+        "name": "Ramit Arora",
```

Comment:
I'll fix the images, I made the mistake of doing it on the judges-page-fb branch instead of fg-responsive